addons/website_slides/tests/test_karma.py

```diff
@@ -78,18 +78,30 @@ def test_karma_gain(self):
 
         # Vote for a slide
         slide_user = self.slide.with_user(user)
+        # up-voting for the first time should add karma
         slide_user.action_like()
         computed_karma += self.channel.karma_gen_slide_vote
         self.assertEqual(user.karma, computed_karma)
-        slide_user.action_like()  # re-like something already liked should not add karma again
-        self.assertEqual(user.karma, computed_karma)
-        slide_user.action_dislike()
+
+        # toggling the up-vote should reduce karma
+        slide_user.action_like()
         computed_karma -= self.channel.karma_gen_slide_vote
         self.assertEqual(user.karma, computed_karma)
+
+        slide_user.action_like()
+        computed_karma += self.channel.karma_gen_slide_vote
+        self.assertEqual(user.karma, computed_karma)
+
+        # down-voting the content that was already up-voted
+        # should reduce the karma two times
         slide_user.action_dislike()
-        computed_karma -= self.channel.karma_gen_slide_vote
+        computed_karma -= (self.channel.karma_gen_slide_vote * 2)
```

Comment:
Useless `()`

Reply:
Hello @std-odoo 
agree but for checking computed karma points we need this.

Reply:
What do you mean ? It's the same
```suggestion
        computed_karma -= self.channel.karma_gen_slide_vote * 2
 ```

Reply:
done!